bindings/rascal/models/IP_generic_md.py

```diff
@@ -0,0 +1,103 @@
+import ase.io
+import ase.units
+import numpy as np
+
+from ..utils import BaseIO, load_obj
+from ..neighbourlist.structure_manager import AtomsList, unpack_ase
+
+
+class GenericMDCalculator(BaseIO):
+
+    """Generic MD driver for a librascal model
+
+    Initialize with model JSON and a structure template, and calculate
+    energies and forces based on position/cell updates _assuming the
+    order and identity of atoms does not change_.
+    """
+
+    def __init__(self, model_json, structure_template, assume_pbc=True):
+        """Initialize a model and structure template
+
+        Parameters
+        ----------
+        model_json  Filename for a JSON file defining the potential
+        structure_template
+                    Filename for an ASE-compatible Atoms object, used
+                    only to initialize atom types and numbers
```

Comment:
How about taking a `species` array/list of integer here instead? This would remove the need to write a file just to communicate atomic species.

Reply:
That would be more of a pain for the user, actually. Many MD codes (including i-PI, and also LAMMPS) take a structure file for initialization of system parameters like the number and type of atoms, the topology, and sometimes other things. It's easiest to just pass the same structure file you used for initialization and let the code extract the information from there.

Reply:
It is quite easy to write a efficient version taking a file as input if we have a version taking an array, but the reverse is not true. Since this is Python, could we do a `isintance()` check? If we have a string, use it as a path; else use it as an array of species.

Reply:
I would actually prefer a more explicit version: Take either a structure template, or a species array, and if neither is provided then complain. Sound good?

Reply:
As keyword arguments? Yes, that sounds good!